src/btree/bt_sync_obsolete.c

```diff
@@ -8,21 +8,25 @@
 
 #include "wt_internal.h"
 
-#define WT_CHECKPOINT_CLEANUP_FILE_INTERVAL 1 /* 1 second */
+#define WT_CC_FILE_INTERVAL 1 /* 1 second */
```

Comment:
Can we have a comment saying what it is for?

Reply:
Done.